More photo

artipavl opened the pull request:
Зробив слайдер

artipavl committed:
переназиває два зображенні

artipavl committed:
Додає розмітку зображень

artipavl committed:
Додає базову розмітку для зображень

artipavl committed:
Merge branch 'main' into more-photo

artipavl committed:
Додає розміри для галереї

artipavl committed:
Merge branch 'main' into more-photo

artipavl committed:
Змінює медіа правило на (min-width: 1440px)

artipavl committed:
Додає стилі для іконки завантаження

artipavl committed:
Додає слайдер

artipavl committed:
Додає стилі до слайдеру

artipavl committed:
Додає скрипт для визову слайдера

artipavl committed:
Додає код для роботи кнопок навігації по слайдеру

artipavl committed:
Додає клас до зображень та кнопки навігації для слайдера

artipavl committed:
Додає стилі для кнопок навігації слайдера

artipavl committed:
Додає скрипт що дозволяє вибрати ібільше зображення

artipavl committed:
Додає альтернативні зображення для слайдеру

artipavl committed:
Додає адаптацію для зображень слайдеру

artipavl committed:
Merge branch 'main' into more-photo

artipavl committed:
Виправляє розмір кнопок навігації слайдеру

artipavl committed:
Додає адаптацію кнопкам навігації слайдеру

artipavl committed:
Додає alt до зображень

artipavl committed:
Виправляє клас показу зображень

artipavl committed:
Додає скрипт для книпки показу зображень

artipavl committed:
Додає зміну типу курсору для зображення

artipavl committed:
Додає сурсор для інтерактивних елементів

artipavl committed:
Додає коментарії для js

artipavl committed:
Додає адаптацію зображень

artipavl committed:
Update more-photo.html

Grimdot reviewed (commented):
Використати міксини в медіазапитах, міксини оголошені в sass/utils/mixins

artipavl committed:
Додає міксини до стилів

artipavl committed:
Додає Masonry Layout для desktop: 1440px

 Masonry Layout для мобільної адекватно зробити не можливо та біда з пропорціями зображень у макеті
